server/sonar-web/src/main/js/helpers/urls.js

```diff
@@ -23,7 +23,7 @@
  * @returns {string}
  */
 export function getComponentUrl (componentKey) {
-  return '/dashboard?id=' + encodeURIComponent(componentKey);
+  return window.baseUrl + '/dashboard?id=' + encodeURIComponent(componentKey);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)